server.py

```diff
@@ -64,6 +64,9 @@
 tournament_put_parser.add_argument('regions', type=list)
 tournament_put_parser.add_argument('pending', type=bool)
 
+smashGgMap_get_parser = reqparse.RequestParser()
```

Comment:
let's change this to smashgg_map_get_parser or smash_gg_map_get_parser to be consistent